x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/BreakingBytesRefBuilder.java

```diff
@@ -25,12 +25,27 @@ public class BreakingBytesRefBuilder implements Accountable, Releasable {
     private final CircuitBreaker breaker;
     private final String label;
 
+    /**
+     * Counts how often we needed to resize while growing. Only works if assertions are enabled.
+     */
+    private int resizes = 0;
```

Comment:
I prefer not to have these 4 bytes to save memory. It's fine not to test the number of resizes. Or we can track these numbers with a custom circuit breaker in tests.

Reply:
Custom circuit breaker is a great idea, I'll go with that!